cv/qc-cv.obo

```diff
@@ -42,336 +40,330 @@ name: has_tuple
 id: has_units
 name: has_units
 
+
+[Term]
+id: MS:4000000
+name: PSI-MS CV Quality Control Vocabulary
+def: "PSI-MS CV Quality Control Vocabulary." [PSI:MS]
+is_a: MS:0000000 ! Proteomics Standards Initiative Mass Spectrometry Vocabularies
```

Comment:
minor observation: so far never used with is_a
https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L9565

Reply:
Thanks, good point. We'll have to fix that.